src/Actions/Misc/ExtensionMethods.cs

```diff
@@ -40,7 +40,7 @@ public static bool NeedNewDataContext(this ElementDataContext dc, DataContextMod
         /// </summary>
         /// <param name="position">Pixel location</param>
         /// <returns>element overlapping point with smallest area, or null if no elements overlap the given point</returns>
-        public static A11yElement GetSmallestElementFromPoint(Dictionary<int, A11yElement> allElements, System.Drawing.Point position)
+        internal static ICoreA11yElement GetSmallestElementFromPoint(Dictionary<int, ICoreA11yElement> allElements, System.Drawing.Point position)
```

Comment:
>internal [](start = 8, length = 8)

ICoreA11yElement is internal at the moment, so this function had to be internal as well.